brightwind/export/export.py

```diff
@@ -154,3 +154,97 @@ def export_csv(data, file_name=None, folder_path=None, **kwargs):
+
+    :param eastings: Eastings of the measurement location in meters.
+    :type eastings: float
+    :param northings: Northings of the measurement location in meters.
+    :type northings: float
+    :param height: Height that the timeseries represents in meters.
+    :type height: float
+    :param wspd_series: Series of wind speed variable
+    :type wspd_series: pandas.Series
+    :param direction_series: Series of wind directions between [0-360]
+    :type direction_series: pandas.Series
+    :param wspd_std_series: Series of wind speed standard deviations
+    :type wspd_std_series: pandas.Series
+    :param site_name: The site name to include in the file.
+           i.e 'Demo Data'
+    :type site_name: str
+    :param file_name: The file name under which the tab file will be saved, or use the default,
+        i.e '2019-06-07_brightwind_tws_export.tab'
+    :type file_name: str
+    :param folder_path: The directory where the tab file will be saved, default is the working directory.
+    :type folder_path: str
+
+    :return: Creates a WindSim timeseries tws file which can be used in the WindSim software.
+
+    **Example Usage**
+    ::
+        import brightwind as bw
+        df = bw.load_campbell_scientific(bw.demo_datasets.demo_campbell_scientific_site_data)
+
+        bw.export_tws_file(df.Spd80mN, df.Dir78mS, 'Demo Data', 80, file_name='campbell_tws_file', 626100, 827971, folder_path=r'C:\some\folder\')"""
+    
+    if site_name is None:
+        site_name = 'brightwind_site'
+    
+    if file_name is None:
+        file_name = str(datetime.datetime.now().strftime("%Y-%m-%d")) + '_brightwind_tws_export.tws'
+
+    if folder_path is None:
+        folder_path = os.getcwd()
+
+    if '.tws' not in file_name:
+        file_name = file_name + '.tws'
+
+    include_sd = True if wspd_std_series is not None else False
+    
+    file_name_print = os.path.splitext(file_name)[0]
+    file_path = os.path.join(folder_path, file_name)
+
+    eastings, northings = int(eastings), int(northings)
+    
+    local_wspd_series = brightwind.analyse.analyse._convert_df_to_series(wspd_series.copy())
+    local_direction_series = brightwind.analyse.analyse._convert_df_to_series(direction_series.copy())
+    if include_sd: local_wspd_sd_series = brightwind.analyse.analyse._convert_df_to_series(wspd_std_series.copy())
+
+    current_timestamp = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    version = brightwind.__version__
+    
+    tws_string = f"{str(file_name_print)} created using brightwind version {version} at {current_timestamp}.\n"
+
+    tws_output = pd.concat([round(local_direction_series,4),round(local_wspd_series,4)],axis=1, keys=['dir:','speed:']).dropna()
+    if include_sd: tws_output = pd.concat([round(local_direction_series,4),round(local_wspd_series,4), round(local_wspd_sd_series,4)],axis=1, keys=['dir:','speed:','SDspeed:']).dropna()
```

Comment:
As `tws_output` is already created you can add the stddev series to it, instead of overwriting it.

Reply:
Updated.